lib/tools/android-manifest.js

```diff
@@ -89,18 +89,6 @@ manifestMethods.packageAndLaunchActivityFromManifest = async function (localApk)
   }
 };
 
-manifestMethods.targetSdkVersionFromManifest = async function (localApk) {
```

Comment:
I'd rather keep it. This method should be simply fixed by checking the the _android:minSdkVersion_ value (it is mandatory to set) in case _ targetSdkVersion_ is omitted in the manifest file. Check https://developer.android.com/guide/topics/manifest/uses-sdk-element.html for more details.